docs/hotfix.md

```diff
@@ -0,0 +1,70 @@
+# How to make a hotfix release
+
+This document describes how to generate a hotfix release for backplane. 
+
+### When to make a hotfix release
+
+- A critical bug has been found
+- Security vulnerabilities have been encountered
+- Something broke in the last/latest release
+
+### Hotfix release process technical guidelines
+
+Please make sure to adhere to these following guidelines below for the hotfix release process : 
+
+#### 1. Create an OSD card
+
+- Create an OSD card with title : **Backplane Hotfix Release for <$reason>**
+- In the description of the card , mention details about the intention/reason behind this hotfix.
+- Add backplane-hotfix label to it.
+
+#### 2. Create pull request 
+
+The main aim is to put hotfixes to production without promoting all changes from the main branch.
+
+- Create a branch called **hotfix** in your code repository. 
+NOTE: This branch should be created from the commit that is currently deployed to production.
+
+- ##### If the hotfix is on reverting a commit : 
```

Comment:
As backplane maintainers, we have permission to use GitHub's UI to revert a commit by clicking the `revert` button (It will auto-create a revert PR which might be easier)
![image](https://github.com/user-attachments/assets/bfedd1e6-411f-4207-a294-791d9ecd4407)

so, may be add alternative phrase like:
```
Alternatively, you can get backplane maintainers to use GitHub's UI to revert a commit directly from the merged PR, which simplifies the process and avoids potential errors in command execution.
```